controllers/nutanixcluster_controller.go

```diff
@@ -50,9 +50,10 @@ import (
 
 // NutanixClusterReconciler reconciles a NutanixCluster object
 type NutanixClusterReconciler struct {
-	Client         client.Client
-	SecretInformer coreinformers.SecretInformer
-	Scheme         *runtime.Scheme
+	Client            client.Client
+	SecretInformer    coreinformers.SecretInformer
+	ConfigMapInformer coreinformers.ConfigMapInformer
```

Comment:
its unclear to me why this new dependency on cmInformer is required?

Reply:
To fetch the trust bundle from the configmap